src/components/CatalogCard/CatalogCard.tsx

```diff
@@ -59,31 +70,42 @@ export const CatalogCard: FunctionComponent<CatalogCardProps> = ({
   const languages = pkg.languages ?? {};
   const targets = Object.keys(languages) as Language[];
 
+  const getUrl = (params?: Partial<Parameters<typeof getPackagePath>[0]>) =>
+    getPackagePath({
+      name: pkg.name,
+      version: pkg.version,
+      language: currentLanguage,
+      ...params,
+    });
+
   return (
-    <CatalogCardContainer url={url}>
+    <CatalogCardContainer isLink>
       <Stack maxH="100%" maxW="100%" overflow="hidden" p={4} spacing={2}>
         {/* Name & Version */}
-        <Text
-          color="blue.800"
-          data-testid={testIds.name}
-          fontWeight="semibold"
-          isTruncated
-        >
-          {pkg.name}
-
+        <LinkOverlay as={Link} to={getUrl()}>
           <Text
-            as="span"
-            color="blue.500"
-            data-testid={testIds.version}
-            fontSize="xs"
-            ml={2}
+            color="blue.800"
+            data-testid={testIds.name}
+            fontWeight="semibold"
+            isTruncated
           >
-            {pkg.version}
+            {pkg.name}
+
+            <Text
+              as="span"
```

Comment:
Can't all these font/color/size definitions be implemented with css class?

Reply:
We don't use plain stylesheets, we leverage a styled system from https://chakra-ui.com/docs/principles